packages/cli/src/validatorDir/ValidatorDir.ts

```diff
@@ -71,6 +72,10 @@ export class ValidatorDir {
     }
   }
 
+  static async create(baseDir: string, pubkey: string, options?: IValidatorDirOptions): Promise<ValidatorDir> {
+    return new ValidatorDir(baseDir, pubkey, options);
```

Comment:
What the reason for an async function that instantiates the ValidatorDir class? It turns a bunch of functions async downstream

Reply:
whoops, this should be removed